src/Form/LegacyConsumer/Commands/SetupFieldValidation.php

```diff
@@ -45,6 +47,18 @@ public function __invoke( $hook ) {
 		$collection->walkFields(
 			/* @var File $field */
 			function( $field ) {
+				/**
+				 * Use this filter to validate custom field which does not exist in field api.
+				 *
+				 * @unreleased
+				 *
+				 * @param Field $field
+				 * @param int $formId
+				 */
+				if( apply_filters( "give_fields_api_validate_{$field->getType()}", false, $field, $this->formId ) ) {
+					return;
+				}
```

Comment:
Let's make use of the `Types::all()` method to avoid the "return true to stop" paradigm. We can do something like:
```php
if ( $field->getType() === Types::FILE ) {
  // Handle file field
} elseif( in_array($field->getType(), Types::all(), true) ) {
  // Handle all other fields
} else {
  do_action("give_fields_api_validate_{$field->getType()}", $field, $this->formId);
}
```
I'm not a fan of the "return to stop" paradigm as it's prone to error.

Reply:
@JasonTheAdams Your suggested changes have been implemented [here](https://github.com/impress-org/givewp/pull/5944/files/b5aa2e3eb80e5aa3452300d56b1623a1e468dd27..9c5b539b0b8ed83a5dcdf4b0c520de0cbe6b0669) (along with a few other changes in regards to docblocks and formatting.